src/languages/en.ts

```diff
@@ -3291,6 +3291,55 @@ export default {
     subscription: {
         mobileReducedFunctionalityMessage: 'You can’t make changes to your subscription in the mobile app.',
         billingBanner: {
+            policyOwnerAmountOwed: {
+                title: 'Your payment info is outdated',
+                subtitle: ({date}) => `Update your payment card by ${date} to continue using all of your favorite features.`,
+            },
+            policyOwnerAmountOwedOverdue: {
+                title: 'Your payment info is outdated',
+                subtitle: 'Please update your payment information.',
+            },
+            policyOwnerUnderInvoicing: {
+                title: 'Your payment info is outdated',
+                subtitle: ({date}) => `Your payment is past due. Please pay your invoice by ${date} to avoid service interruption.`,
+            },
+            policyOwnerUnderInvoicingOverdue: {
+                title: 'Your payment info is outdated',
+                subtitle: 'Your payment is past due. Please pay your invoice.',
+            },
+            billingDisputePending: {
+                title: 'Your card couldn’t be charged',
+                subtitle: ({amountOwed, cardEnding}) =>
+                    `You disputed the ${amountOwed} charge on the card ending in ${cardEnding}. Your account will be locked until the dispute is resolved with your bank.`,
+            },
+            cardAuthenticationRequired: {
+                title: 'Your card couldn’t be charged',
+                subtitle: ({cardEnding}) =>
+                    `Your payment card hasn’t been fully authenticated. Please complete the authentication process to activate your payment card ending in ${cardEnding}.`,
+            },
+            insufficientFunds: {
+                title: 'Your card couldn’t be charged',
+                subtitle: ({amountOwed}) =>
+                    `Your payment card was declined due to insufficient funds. Please retry or add a new payment card to clear your ${amountOwed} outstanding balance.`,
+            },
+            cardExpired: {
+                title: 'Your card couldn’t be charged',
+                subtitle: ({amountOwed}) => `Your payment card expired. Please add a new payment card to clear your ${amountOwed} outstanding balance.`,
+            },
+            cardExpireSoon: {
+                title: 'Your card is expiring soon',
+                subtitle: 'Your payment card will expire at the end of this month. Click the three-dot menu below to update it and continue using all your favorite features.',
```

Comment:
@blimpich @JKobrynski We're always saying the card will expire at the end of this month irrespective of whether card will expire this month or next month. Do we want to update this?

Reply:
How difficult is fixing this? If its going to delay this PR I say we just make a follow up polish issue for it. If this is an easy fix then lets fix it now. Do we just need the copy?

Reply:
The code logic would be fairly simple, I think it's just about comparing dates and mixing that with some if statements, that would be on my end. What I'd need from you is the exact text we want to display for each scenario (both English and Spanish) and some guidelines on how we should distinguish these scenarios (e.g. less than X days, display "end of this month" variant, else display "next month" variant). I can do it in this PR or I can create a separate next week if you'd like, up to you!

Reply:
I think we should add a small PR as a follow up while we come up with the text. This has already gone through several reviews.

Reply:
Sure, let's do it that way. Once you create an issue/slack thread, you can assign/tag me right away, and I will take care of it asap.

Reply:
@blimpich can create one and assign both of us.

Reply:
Sorry I haven't been able to create this issue. I have to get the copy / translation first and I haven't been able to prioritize that yet. Will get to this eventually.